### ISSUE TITLE
Krew install does not work on Openshift.

### ISSUE THREAD

**Issue by aaronmcohen** [minio/operator#1051]
installation via Krew on Openshift 4.9 fails.

## Expected Behavior
 `oc minio proxy -n minio-operator` starts the proxy

## Current Behavior
ReplicaSets are unable to scale up due to:
```
Error creating: pods "minio-operator-66bdf84c4b-" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, spec.containers[0].securityContext.runAsUser: Invalid value: 1000: must be in the ranges: [1000690000, 1000699999], provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "noobaa": Forbidden: not usable by user or serviceaccount, provider "noobaa-endpoint": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler": Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, provider "rook-ceph": Forbidden: not usable by user or serviceaccount, provider "node-exporter": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount, provider "rook-ceph-csi": Forbidden: not usable by user or serviceaccount]
```

## Possible Solution
I found similar minio GitHub issues where the runAsUser can't be 1000. However I can't set this from krew.

## Steps to Reproduce (for bugs)
1. oc krew update
2. oc krew install minio
3. oc minio version                 
> v4.4.11
4. oc minio init       5. 
```
namespace/minio-operator created
serviceaccount/minio-operator created
clusterrole.rbac.authorization.k8s.io/minio-operator-role created
clusterrolebinding.rbac.authorization.k8s.io/minio-operator-binding created
customresourcedefinition.apiextensions.k8s.io/tenants.minio.min.io created
service/operator created
deployment.apps/minio-operator created
serviceaccount/console-sa created
clusterrole.rbac.authorization.k8s.io/console-sa-role created
clusterrolebinding.rbac.authorization.k8s.io/console-sa-binding created
configmap/console-env created
service/console created
deployment.apps/console created
-----------------

To open Operator UI, start a port forward using this command:

kubectl minio proxy -n minio-operator 

-----------------
```
6.  oc describe  replicaset -n minio-operator
```yaml
Name:           console-58b7dc8856
Namespace:      minio-operator
Selector:       app=console,pod-template-hash=58b7dc8856
Labels:         app=console
                pod-template-hash=58b7dc8856
Annotations:    deployment.kubernetes.io/desired-replicas: 1
                deployment.kubernetes.io/max-replicas: 2
                deployment.kubernetes.io/revision: 1
                operator.min.io/authors: MinIO, Inc.
                operator.min.io/license: AGPLv3
                operator.min.io/support: https://subnet.min.io
Controlled By:  Deployment/console
Replicas:       0 current / 1 desired
Pods Status:    0 Running / 0 Waiting / 0 Succeeded / 0 Failed
Pod Template:
  Labels:           app=console
                    pod-template-hash=58b7dc8856
  Annotations:      operator.min.io/authors: MinIO, Inc.
                    operator.min.io/license: AGPLv3
                    operator.min.io/support: https://subnet.min.io
  Service Account:  console-sa
  Containers:
   console:
    Image:       minio/console:v0.15.0
    Ports:       9090/TCP, 9433/TCP
    Host Ports:  0/TCP, 0/TCP
    Args:
      server
    Environment:
      CONSOLE_OPERATOR_MODE:  on
    Mounts:                   <none>
  Volumes:                    <none>
Conditions:
  Type             Status  Reason
  ----             ------  ------
  ReplicaFailure   True    FailedCreate
Events:
  Type     Reason        Age                  From                   Message
  ----     ------        ----                 ----                   -------
  Warning  FailedCreate  21s (x15 over 103s)  replicaset-controller  Error creating: pods "console-58b7dc8856-" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, spec.containers[0].securityContext.runAsUser: Invalid value: 1000: must be in the ranges: [1000700000, 1000709999], provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "noobaa": Forbidden: not usable by user or serviceaccount, provider "noobaa-endpoint": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler": Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, provider "rook-ceph": Forbidden: not usable by user or serviceaccount, provider "node-exporter": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount, provider "rook-ceph-csi": Forbidden: not usable by user or serviceaccount]


Name:           minio-operator-66bdf84c4b
Namespace:      minio-operator
Selector:       name=minio-operator,pod-template-hash=66bdf84c4b
Labels:         name=minio-operator
                pod-template-hash=66bdf84c4b
Annotations:    deployment.kubernetes.io/desired-replicas: 2
                deployment.kubernetes.io/max-replicas: 3
                deployment.kubernetes.io/revision: 1
                operator.min.io/authors: MinIO, Inc.
                operator.min.io/license: AGPLv3
                operator.min.io/support: https://subnet.min.io
Controlled By:  Deployment/minio-operator
Replicas:       0 current / 2 desired
Pods Status:    0 Running / 0 Waiting / 0 Succeeded / 0 Failed
Pod Template:
  Labels:           name=minio-operator
                    pod-template-hash=66bdf84c4b
  Annotations:      operator.min.io/authors: MinIO, Inc.
                    operator.min.io/license: AGPLv3
                    operator.min.io/support: https://subnet.min.io
  Service Account:  minio-operator
  Containers:
   minio-operator:
    Image:      minio/operator:v4.4.11
    Port:       <none>
    Host Port:  <none>
    Requests:
      cpu:                200m
      ephemeral-storage:  500Mi
      memory:             256Mi
    Environment:
      CLUSTER_DOMAIN:  cluster.local
    Mounts:            <none>
  Volumes:             <none>
Conditions:
  Type             Status  Reason
  ----             ------  ------
  ReplicaFailure   True    FailedCreate
Events:
  Type     Reason        Age                  From                   Message
  ----     ------        ----                 ----                   -------
  Warning  FailedCreate  22s (x15 over 105s)  replicaset-controller  Error creating: pods "minio-operator-66bdf84c4b-" is forbidden: unable to validate against any security context constraint: [provider "anyuid": Forbidden: not usable by user or serviceaccount, spec.containers[0].securityContext.runAsUser: Invalid value: 1000: must be in the ranges: [1000700000, 1000709999], provider "nonroot": Forbidden: not usable by user or serviceaccount, provider "noobaa": Forbidden: not usable by user or serviceaccount, provider "noobaa-endpoint": Forbidden: not usable by user or serviceaccount, provider "hostmount-anyuid": Forbidden: not usable by user or serviceaccount, provider "machine-api-termination-handler": Forbidden: not usable by user or serviceaccount, provider "hostnetwork": Forbidden: not usable by user or serviceaccount, provider "hostaccess": Forbidden: not usable by user or serviceaccount, provider "rook-ceph": Forbidden: not usable by user or serviceaccount, provider "node-exporter": Forbidden: not usable by user or serviceaccount, provider "privileged": Forbidden: not usable by user or serviceaccount, provider "rook-ceph-csi": Forbidden: not usable by user or serviceaccount]

```

## Your Environment
oc version
```
Client Version: 4.6.17
Server Version: 4.9.15
Kubernetes Version: v1.22.3+e790d7f
```

**Comment by dvaldivia** [comment]
I think this have been fixed, right @pjuarezd @cniackz ?

**Comment by pjuarezd** [comment]
FIxed on Operator v5.0.4
